eclipsing-binary-simulator/src/Window.jsx

```diff
@@ -89,11 +91,17 @@ export default class Window extends React.Component {
     onDrag(e) {
         if (this.state.isDragging) {
             this.setState({
-                x: e.pageX - this.state.xDiff,
-                y: e.pageY - this.state.yDiff
+                x: this.clamp(e.pageX - this.state.xDiff),
+                y: this.clamp(e.pageY - this.state.yDiff)
             });
         }
     }
+    clamp(n) {
```

Comment:
@nikolas does `clamp` need to be bound to `this`?

Reply:
It doesn't. I just moved it, thanks.